### ISSUE TITLE
Pro Feature upsell cards - update text color at bottom of cards for accesibility compliance

### ISSUE THREAD

**Issue by nicolepaschen** [BoldGrid/w3-total-cache#827]
Right now these cards have white text at the bottom, which does not pass accessibility contrast requirements. Please update the text to black like shown in this screenshot:

![change-text-to-black](https://github.com/BoldGrid/w3-total-cache/assets/20601160/e2ff9242-2755-41b7-ae5f-1417d02db0f0)
